### PR TITLE
Amanda lane

### PR DESCRIPTION
**Pull request by amlane**

### CODE REVIEW

**Comment by amlane** [comment]
Adding reviewer

File: students/students-model.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+function update(id, changes) {
+    return db('students')
+    .where({ id })
+    .update(changes)
+};
+
+function remove(id) {
+    return db('students')
+    .where({ id })
+    .delete();
```
Comment (BrannanC) [inline students/students-model.js:39]: delete is a reserved word in js so it's better to use .del()

File: students/students-router.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    })
+    } 
+});
+
+router.delete('/:id', verifyId, (req, res) => {
+    const id = req.params.id;
+
+    Students.remove(id)
+    .then(deletedStudent => {
+        res.status(200).json({ message: `Student expelled! Oh noooo!` });
```
Comment (BrannanC) [inline students/students-router.js:65]: Yikes

**Review by BrannanC — COMMENTED** [review]
- Super clean, I have no real improvements for you
- At some point you're going to want to switch over to async/await

## Rating (1-3)
3
****

**Pull request merged**